src/libraries/Native/Unix/System.Native/pal_process.c

```diff
@@ -304,7 +304,11 @@ int32_t SystemNative_ForkAndExecProcess(const char* filename,
     // Process is still the clone of this one. This is a best-effort attempt, so ignore any errors.
```

Comment:
This comment says the code is intentionally ignoring errors. Is it a good idea to add the error handling here?

Reply:
Yes, errors should be ignored.  I have fixed this.